### DISCUSSION TITLE
Axis-aligned box

### DISCUSSION THREAD

**Discussion by ghost** [STVIR/pysot#240] [General]
the result is : Axis-aligned box ,but ground truth is not Axis-aligned box ,
how to visualiasation it ?
thanks

**Comment by ZhiyuanChen** [comment]
Sorry but I am having troubles to understand your question.
You may use Chinese or other languages if you find it's easier.

**Comment by ZhiyuanChen** [comment]
Since no further question is asked, I'm closing it for now.
Please reopen it or create a new issue if you have further question.
